orttraining/orttraining/test/python/perf_log/ort_module_perf_test_tools.py

```diff
@@ -0,0 +1,231 @@
+# https://docs.microsoft.com/en-us/azure/mysql/connect-python
+
+import mysql.connector
+from mysql.connector import errorcode
+import git
+import os
+
+import argparse
+from datetime import datetime
+
+def get_repo_commit(repo_path):
+    repo = git.Repo(repo_path, search_parent_directories=True)
+    sha = repo.head.object.hexsha
+    short_sha = repo.git.rev_parse(sha, short=4)
+    return short_sha
+
+create_table_script = "CREATE TABLE perf_test_training_ort_module_data (\
+    id int(11) NOT NULL AUTO_INCREMENT,\
+    Model varchar(64) COLLATE utf8_bin DEFAULT NULL,\
+    BatchId varchar(32) COLLATE utf8_bin DEFAULT NULL,\
+    CommitId varchar(32) COLLATE utf8_bin DEFAULT NULL,\
+    ModelName varchar(256) COLLATE utf8_bin DEFAULT NULL,\
+    DisplayName varchar(512) COLLATE utf8_bin DEFAULT NULL,\
+    UseMixedPrecision tinyint(1) DEFAULT NULL,\
+    UseAutoCast tinyint(1) DEFAULT NULL,\
+    UseDeepSpeed tinyint(1) DEFAULT NULL,\
+    Optimizer varchar(32) COLLATE utf8_bin DEFAULT NULL,\
+    BatchSize int(11) DEFAULT NULL,\
+    SeqLen int(11) DEFAULT NULL,\
+    PredictionsPerSeq int(11) DEFAULT NULL,\
+    NumOfBatches int(11) DEFAULT NULL,\
+    WeightUpdateSteps int(11) DEFAULT NULL,\
+    Round int(11) DEFAULT NULL,\
+    GradAccSteps int(11) DEFAULT NULL,\
+    AvgTimePerBatch float DEFAULT NULL,\
+    Throughput float DEFAULT NULL,\
+    StabilizedThroughput float DEFAULT NULL,\
+    EndToEndThroughput float DEFAULT NULL,\
+    TotalTime float DEFAULT NULL,\
+    AvgCPU int(11) DEFAULT NULL,\
+    Memory int(11) DEFAULT NULL,\
+    RunConfig varchar(2048) COLLATE utf8_bin DEFAULT NULL,\
+    Time datetime DEFAULT NULL,\
+    PRIMARY KEY (id),\
+    UNIQUE KEY config_unique (Model,BatchId,CommitId,UseMixedPrecision,UseAutoCast,UseDeepSpeed,Optimizer,BatchSize,SeqLen,ModelName)\
+) ENGINE=InnoDB AUTO_INCREMENT=1696 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;"
+
+insert_table_script = "INSERT INTO onnxruntime.perf_test_training_ort_module_data\
+    (\
+    Model,\
+    BatchId,\
+    CommitId,\
+    ModelName,\
+    DisplayName,\
+    UseMixedPrecision,\
+    UseAutoCast,\
+    UseDeepSpeed,\
+    Optimizer,\
+    BatchSize,\
+    SeqLen,\
+    PredictionsPerSeq,\
+    NumOfBatches,\
+    WeightUpdateSteps,\
+    Round,\
+    GradAccSteps,\
+    AvgTimePerBatch,\
+    Throughput,\
+    StabilizedThroughput,\
+    EndToEndThroughput,\
+    TotalTime,\
+    AvgCPU,\
+    Memory,\
+    RunConfig,\
+    Time)\
+VALUES\
```

Comment:
was the indentation intentionally done this way or was it unintentional?

Reply:
perhaps we should indent the values below

---
In reply to: [587699060](https://github.com/microsoft/onnxruntime/pull/6806#discussion_r587699060) [](ancestors = 587699060)

Reply:
indent corrected